.github/workflows/pull-request.yaml

```diff
@@ -48,6 +48,26 @@ jobs:
           echo "$PROD_DIFF" >> $GITHUB_ENV
           echo "EOF" >> $GITHUB_ENV
 
+      - name: Create nginx conf file
+        env:
+          NGINX_SERVER_NAME: epoc-frontend.run.app
+        run: envsubst '$NGINX_SERVER_NAME' < nginx.conf.tmpl > nginx.conf
+      
+      -  name: Set up Docker Buildx
+         uses: docker/setup-buildx-action@v2
+
+      - name: Build and push image to artifact registry
+        uses: docker/build-push-action@v4
+        with:
+          build-args: |
+            NEXT_PUBLIC_FIREBASE_API_KEY=asd
+            NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=asd
+            NEXT_PUBLIC_API_URL=http://localhost
+            NEXT_PUBLIC_GOOGLE_APIKEY=asd
+          context: .
+          file: Dockerfile
+          no-cache: false
+          push: false
```

Comment:
Tämän buildin testaamisen voi siirtää kokonaan ennen tuota `Check types in dev` steppiä. Nyt tää on vähän hassusti keskellä noita tyyppitarkistuksia